app/controllers/admin/employees_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Admin::EmployeesController < ApplicationController
+
+  def index
+    @stores = User.find(current_user.id).stores
+  end
+
+  def update
+    UserRole.admin_update(params[:id], params[:store_id], params[:fire])
```

Comment:
What's the rationale for not updating these with strong params?

def user_role_params
params.permit(:id, :store_id, :fire)
end

Reply:
Again! Nice catch! I'll update this as well.

Reply:
I can definitely do that! And this made me think of adding in some kind of scope to protect against a store admin firing or promoting users of other stores (that they aren't associated with)

Reply:
I hadn't considered that scenario. We could create another card for it so that we're aware of it since it feels more like an edge-case (and given the deadline).